openfisca_country_template/parameters/benefits/basic_income.yaml

```diff
@@ -1,4 +1,5 @@
 description: Amount of the basic income
+unit: currency
```

Comment:
Why not `EUR` or `USD` instead of `currency`? These would be proper "units", while `currency` is more vague

Reply:
Another option would be to namespace it : `currency/EUR`.

What do you think @fpagnoux @benjello ?

Reply:
Great minds think alike 😅 https://github.com/openfisca/openfisca-france/pull/1022#MDEyOklzc3VlQ29tbWVudDQwMjg5NTQ5Mg

Reply:
The downside of using `/` though is that it can be confused with a division. I guess we could use `currency:EUR` instead 🤔